### PR TITLE
Adjusted Dialog Styling

### PR DESCRIPTION
**Pull request by CurtisUnderwood**
### Description

List of proposed changes:

- Aligned dialog styling with Figma designs: added background blur and fixed height component

### What to test for/How to test
- Open Quiz Dialog to see changes

### Additional Notes
- Not associated with a story

### CODE REVIEW

File: frontend/src/theme.ts [hunk 1]
```diff
@@ -18,6 +18,13 @@ const theme = createTheme({
     fontFamily: '"Noto Sans", sans-serif',
   },
   components: {
+    MuiDialog:{
+      styleOverrides: {
+        root: {
+          backdropFilter: 'blur(4px)'
+        },
+      },
+    },
```
Comment (sebastien-comeau) [inline frontend/src/theme.ts:27]: Override Backdrop component instead.

```suggestion
    MuiBackdrop: {
      styleOverrides: {
        root: {
          backdropFilter: 'blur(4px)',
        },
      },
    },
```

File: frontend/src/pages/learn/index.tsx [hunk 2]
```diff
@@ -247,9 +265,18 @@ const Learn: FC = () => {
...
+                  sx={{
+                    position: 'absolute',
+                    bottom: 0,
+                    left: 0,
+                    right: 0,
+                    p: 4,
+                  }}>
                     {(currentStepIndex ?? 0) > 0 && (
-                      <>
+                      <div className="">
```
Comment (sebastien-comeau) [inline frontend/src/pages/learn/index.tsx:279]: Use React Fragment shorthand insteadl.
https://react.dev/reference/react/Fragment

```suggestion
                      <>
```

File: frontend/src/pages/learn/index.tsx [hunk 3]
```diff
@@ -247,9 +265,18 @@ const Learn: FC = () => {
                     <h2 className="mb-8 font-display text-2xl font-medium md:mb-16 md:rounded-3xl md:bg-[#f5f5f5] md:p-6 md:text-4xl md:text-primary-700">
                       {t('quiz.navigation.title')}
                     </h2>
-                    <div className="mb-5">{renderComponent()}</div>
+                    <div>{renderComponent()}</div>
+                  </DialogContent>
+                  <DialogActions className="block"
```
Comment (sebastien-comeau) [inline frontend/src/pages/learn/index.tsx:270]: Use `flex` to move the `Stepper` to the bottom of `DialogContent`. `Stepper` shouldn't be inside the `DialogActions` component.

File: frontend/src/pages/learn/index.tsx [hunk 4]
```diff
@@ -68,9 +68,16 @@ const QuizConfirmation: FC<QuizConfirmationProps> = ({
...
       </DialogContent>
-      <DialogActions className="block">
+      <DialogActions className="block"
+      sx={{
+        position: 'absolute',
+        bottom: 0,
+        left: 0,
+        right: 0,
+        p: 4,
+      }}>
```
Comment (sebastien-comeau) [inline frontend/src/pages/learn/index.tsx:80]:
```suggestion
      <DialogActions className="block">
```

File: frontend/src/pages/learn/index.tsx [hunk 5]
```diff
@@ -172,6 +179,17 @@ const Learn: FC = () => {
...
+          '& .MuiDialog-paper': {
+            height: '90%',
+            minHeight: '800px',
+            padding: '10px',
+            '@media (max-width: 960px)': {
+              height: '100%',
+            },
+          }
+        }}
+
```
Comment (sebastien-comeau) [inline frontend/src/pages/learn/index.tsx:192]: Set `className` on the `DialogContent` prop.
```suggestion
```

Comment (sebastien-comeau) [reply frontend/src/pages/learn/index.tsx:192]:
```tsx
<DialogContent className="min-h-[700px]">
```

**Comment by gregory-j-baker** [comment]
FYI, I rebased on `main` and force-pushed.

File: frontend/src/theme.ts (hunk 1 above)
Comment (CurtisUnderwood) [reply frontend/src/theme.ts:27]: Updated

File: frontend/src/pages/learn/index.tsx (hunk 2 above)
Comment (CurtisUnderwood) [reply frontend/src/pages/learn/index.tsx]: Fixed

**Comment by gregory-j-baker** [comment]
Can the associated branch for this be deleted?

**Pull request closed without merging**